index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
- [ ] Kindly make sure your website title matches that of the website. People who visit your website will know what website they are visiting.

Reply:
@kingsleyocran  it would be better if you try to give an understandable with professional English  comment in order for the student to understand what you've required them to change  
for this part 
![image](https://user-images.githubusercontent.com/61790587/171041119-e5e3aa3d-65d5-4ba7-9776-435adbe21fcb.png)
not clear enough so please try to correct that

Reply:
@kingsleyocran Good job implementing the changes :+1: